etl/steps/data/garden/state_capacity/2023-10-19/state_capacity_dataset.meta.yml

```diff
@@ -0,0 +1,447 @@
+          child mortality, gender equality, HIV/AIDS, immunizations, income poverty, maternal health,
+          per capita GDP growth, and primary school completion.
+
+
+          We have divided the resulting measure by 100 so that 1 is the highest possible value. The data
+          cover 127 countries from 2004 to 2015.
+        processing_level: minor
+        display:
+          name: Statistical Capacity
+          numDecimalPlaces: 1
+      tax_inc_tax:
+        title: Taxes on income, profits and capital gains (% of taxes)
+        unit: "%"
+        short_unit: "%"
+        description_short: Measures the percentage of total tax revenue that comes from taxes on income, profits, and capital gains.
+        description_from_producer: >-
+          For this measure, we start with the Government Revenue Dataset from the International Centre
+          for Tax and Development (ICTD). We then expand coverage of these data considerably through a
+          variety of sources: International Monetary Fund yearbooks, electronic archives of older IMF data,
+          and a collection of estimates from scholars studying specific countries. Using these source data, we
+          calculate the percentage of total tax revenue that comes from taxes on income, profits, and capital
+          gains. For use in analysis, we rescale this percentage to a proportion. The data cover 1963-2015
+          for 168 countries, with broad coverage beginning in the early 1970s.
+        processing_level: minor
+        display:
+          name: Taxes on income (% of taxes)
+          numDecimalPlaces: 1
+      tax_trade_tax:
+        title: Taxes on international trade (% of taxes)
+        unit: "%"
+        short_unit: "%"
+        description_short: Measures the percentage of total tax revenue that comes from taxes on international trade.
+        description_from_producer: >-
+          For this measure, we start with the Government Revenue Dataset from the International Centre
+          for Tax and Development (ICTD). We then expand coverage of these data considerably through a
+          variety of sources: International Monetary Fund yearbooks, electronic archives of older IMF data,
+          and a collection of estimates from scholars studying specific countries. Using these source data,
+          we calculate the percentage of total tax revenue that comes from taxes on international trade. For
+          use in analysis, we rescale this percentage to a proportion. The data cover 1960-2015 for 167
+          countries, with broad coverage beginning in the early 1970s.
+        processing_level: minor
+        display:
+          name: Taxes on international trade (% of taxes)
+          numDecimalPlaces: 1
+      taxrev_gdp:
+        title: Tax revenue (% GDP)
+        unit: "%"
+        short_unit: "%"
+        description_short: Measures tax revenue as a percentage of GDP.
```

Comment:
This could be the title, and no `description_short` would be required. The short description should complement the title, not repeat it.